website/package.json

```diff
@@ -1,55 +1,70 @@
 {
+  "engines": {
+    "node": ">=18.0"
+  },
   "scripts": {
-    "start-server": "./test-links.sh start",
-    "check-links": "./test-links.sh check",
-    "stop-server": "./test-links.sh stop",
     "full-test": "./test-links.sh test",
-    "examples": "docusaurus-examples",
-    "start": "node ./utils/getChangelogs.js && docusaurus start",
-    "build": "node ./utils/getChangelogs.js && docusaurus build",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version",
+    "start": "docusaurus start",
```

Comment:
If we remove `getChangelogs.js` script here we need to implement another way of checking for a new release report.

Reply:
added a script to update it, but didn't add it on every start. We should automatize that with actions maybe.